test/check-machines-migrate

```diff
@@ -90,47 +90,47 @@ class TestMachinesMigration(VirtualMachinesCase):
     def testSharedStorageMigration(self):
         self._testMigrationGeneric(False, False, None)
 
-    @skipImage('RHEL doesnt provide support for copy-storage migration', "rhel-8-4", "rhel-8-5", "rhel-8-6", "rhel-9-0", "centos-9-stream")
+    @skipImage('RHEL does not provide support for copy-storage migration', "rhel-8-4", "rhel-8-5", "rhel-8-6", "rhel-9-0", "centos-9-stream")
```

Comment:
Either `doesn't` needs to be used (and the single tick quotes need to be double tick quotes) or we expand the word to `does not`, which is what I did in this file. I think it's just text to explain what's happening in the test, so it's fine to change, correct? There are several other occurrences in this file.

Reply:
This is fine to change indeed.